sentry-sidekiq/lib/sentry/sidekiq/sentry_context_middleware.rb

```diff
@@ -23,7 +23,15 @@ def call(_worker, job, queue)
         scope.set_contexts(sidekiq: job.merge("queue" => queue))
         scope.set_transaction_name(context_filter.transaction_name, source: :task)
         transaction = start_transaction(scope, job["trace_propagation_headers"])
-        scope.set_span(transaction) if transaction
+
+        if transaction
+          scope.set_span(transaction)
+
+          transaction.set_data("messaging.message.id", job["jid"])
+          transaction.set_data("messaging.destination.name", queue)
+          transaction.set_data("messaging.message.receive.latency", ((Time.now.to_f - job["enqueued_at"]) * 1000).to_i)
```

Comment:
Are we sure that both time objects will use the same timezone?

Reply:
Should be - enqueued_at is generated with `Time.now.to_f` as well in Sidekiq.